iodata/formats/wfx.py

```diff
@@ -56,8 +56,10 @@ def load_data_wfx(lit: LineIterator) -> dict:
         '<Number of Alpha Electrons>': 'num_alpha_electron',
         '<Number of Beta Electrons>': 'num_beta_electron',
         '<Number of Primitives>': 'num_primitives',
-        '<Electronic Spin Multiplicity>': 'spin_multi',
```

Comment:
This is a minor remark, but here it is better to keep the `,` after the last element of a sequence. In this way, when someone adds a new element to the sequence, the last line doesn't need to change for adding the `,`. This makes the changes shown by git more meaningful.

Reply:
I never thought about this, but this is definitely necessary. I have followed this pattern for the coming commits. Thank you for pointing it out and I learned something new.